test/common/alerts_mechanism.go

```diff
@@ -31,73 +26,79 @@ type alertManagerConfig struct {
 }
 
 const (
-	threescaleOperatorDeploymentName          = "3scale-operator"
-	threescaleApicastProdDeploymentConfigName = "apicast-production"
-	monitoringTimeout                         = time.Minute * 25
-	monitoringRetryInterval                   = time.Minute
-	verifyOperatorDeploymentTimeout           = time.Minute * 5
-	verifyOperatorDeploymentRetryInterval     = time.Second * 15
+	keycloakOperatorDeploymentName        = "keycloak-operator"
+	monitoringTimeout                     = time.Minute * 20
+	monitoringRetryInterval               = time.Minute * 1
+	verifyOperatorDeploymentTimeout       = time.Minute * 5
+	verifyOperatorDeploymentRetryInterval = time.Second * 10
 )
 
-var threescaleAlertsToTest = map[string]string{
-	"RHMIThreeScaleApicastProductionServiceEndpointDown": "none",
-	"ThreeScaleApicastProductionPod":                     "none",
+var keycloakAlertsToTest = map[string]string{
+	"RHMIUserRhssoKeycloakOperatorMetricsServiceEndpointDown": "none",
 }
 
 // TestIntegreatlyAlertsMechanism verifies that alert mechanism works
 func TestIntegreatlyAlertsMechanism(t TestingTB, ctx *TestingContext) {
 
-	originalOperatorReplicas, err := getNumOfReplicasDeployment(threescaleOperatorDeploymentName, ctx.KubeClient)
+	originalOperatorReplicas, err := getNumOfReplicasDeployment(keycloakOperatorDeploymentName, ctx.KubeClient)
 	if err != nil {
 		t.Errorf("failed to get number of replicas: %s", err)
 	}
 
 	// verify that alert to be tested is not firing before starting the test
-	err = getThreescaleAlertState(ctx)
+	err = getKeycloakAlertState(ctx)
 	if err != nil {
 		t.Fatal("failed to get threescale alert state", err)
 	}
 
-	threescaleAlertsFiring := false
+	keycloakAlertsFiring := false
 
-	for threescaleAlertName, threescaleAlertState := range threescaleAlertsToTest {
-		if threescaleAlertState != "none" {
-			threescaleAlertsFiring = true
-			t.Errorf("%s alert should not be firing", threescaleAlertName)
+	for keycloakAlertName, keycloakAlertState := range keycloakAlertsToTest {
+		if keycloakAlertState != "none" {
+			keycloakAlertsFiring = true
+			t.Errorf("%s alert should not be firing", keycloakAlertName)
 		}
 	}
 
-	if threescaleAlertsFiring {
-		t.Log("3Scale alerts firing unexpectedly")
+	if keycloakAlertsFiring {
+		t.Log("Keycloak alerts firing unexpectedly")
```

Comment:
If I am understanding correctly at this point we don't expect to see alerts firing. Something that might be more clear in the logs is `Keycloak alerts firing unexpectedly before deployments are scaled down`